src/include/ipc4/base-config.h

```diff
@@ -1,6 +1,6 @@
-/* SPDX-License-Identifier: BSD-3-Clause
- *
- * Copyright(c) 2021 Intel Corporation. All rights reserved.
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
```

Comment:
We dont need to change the date here if we are moving headers from A -> B

Reply:
I always try to update the date when I edit the file. In this case, I cut most of the content, but also added a one new line.